keyboards/handwired/swiftrax/pandamic/keymaps/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This file should be deleted.

Reply:
so do I delete this info.json?

Reply:
This one. `info.json` doesn't do anything at keymap-level, only at keyboard/revision-level.